client/components/infinite-list/index.jsx

```diff
@@ -161,6 +161,27 @@ module.exports = React.createClass( {
 		}
 	},
 
+	reset: function() {
```

Comment:
i do not all this duplication with componentWillMount, but initial attempts to DRY it out made it worse. The state depending on a call to the scrollHelper which is being created here complicates it.

Part of me wants to ditch scrollHelper and yank it back into index. Breaking it out just makes life more complicated.